packages/jest-config/src/__tests__/normalize.test.ts

```diff
@@ -161,65 +161,8 @@ describe('automock', () => {
   });
 });
 
-describe('collectCoverageOnlyFrom', () => {
-  it('normalizes all paths relative to rootDir', async () => {
-    const {options} = await normalize(
-      {
-        collectCoverageOnlyFrom: {
-          'bar/baz': true,
-          'qux/quux/': true,
-        },
-        rootDir: '/root/path/foo/',
-      },
-      {} as Config.Argv,
-    );
-
-    const expected = Object.create(null);
-    expected[expectedPathFooBar] = true;
-    expected[expectedPathFooQux] = true;
-
-    expect(options.collectCoverageOnlyFrom).toEqual(expected);
-  });
-
-  it('does not change absolute paths', async () => {
-    const {options} = await normalize(
-      {
-        collectCoverageOnlyFrom: {
-          '/an/abs/path': true,
-          '/another/abs/path': true,
-        },
-        rootDir: '/root/path/foo',
-      },
-      {} as Config.Argv,
-    );
-
-    const expected = Object.create(null);
-    expected[expectedPathAbs] = true;
-    expected[expectedPathAbsAnother] = true;
-
-    expect(options.collectCoverageOnlyFrom).toEqual(expected);
-  });
-
-  it('substitutes <rootDir> tokens', async () => {
-    const {options} = await normalize(
-      {
-        collectCoverageOnlyFrom: {
-          '<rootDir>/bar/baz': true,
-        },
-        rootDir: '/root/path/foo',
-      },
-      {} as Config.Argv,
-    );
-
-    const expected = Object.create(null);
-    expected[expectedPathFooBar] = true;
-
-    expect(options.collectCoverageOnlyFrom).toEqual(expected);
-  });
-});
-
 describe('collectCoverageFrom', () => {
-  it('substitutes <rootDir> tokens', async () => {
+  it('ignores <rootDir> tokens', async () => {
```

Comment:
Could not leave it. In this case the tokens are ignored. See the test or code here: https://github.com/facebook/jest/blob/3bef02e5925948b38f035572121a41ec9b633dfa/packages/jest-config/src/normalize.ts#L301-L305